### PR TITLE
Python 3: Update dict.items() and dict.values()

### PR DESCRIPTION
**Pull request by hereischen**
Signed-off-by: Haotong Chen <hachen@redhat.com>

### CODE REVIEW

File: aexpect/client.py [hunk 1]
```diff
@@ -187,7 +187,7 @@ def __init__(self, command=None, a_id=None, auto_close=False, echo=False,
         self.reader_fds = {}
         try:
             assert(is_file_locked(self.lock_server_running_filename))
-            for reader, filename in self.reader_filenames.items():
+            for reader, filename in list(self.reader_filenames.items()):
```
Comment (clebergnu) [inline aexpect/client.py]: Why is this needed I don't see the reason for turning the `dict_items` (in Python 3) into a `list`. The former even performs better being working as an iterator.

Can you elaborate on the reason for this change?

File: aexpect/client.py [hunk 2]
```diff
@@ -256,7 +256,7 @@ def _close_reader_fds(self):
         """
         Close all reader file descriptors.
         """
-        for fd in self.reader_fds.values():
+        for fd in list(self.reader_fds.values()):
```
Comment (clebergnu) [inline aexpect/client.py]: Same here.

File: aexpect/client.py (hunk 1 above)
Comment (hereischen) [reply aexpect/client.py]: sorry, I was just think this will work for both python 2 and 3. But I was forgetting it is compatible.
Will close this patch. Thanks.
